color/README.md

```diff
@@ -26,6 +26,46 @@ See https://linebender.org/blog/doc-include/ for related discussion. -->
 
 Color is a Rust crate which implements color space conversions, targeting at least CSS4 color.
 
+## Scope and goals
+
+Color in its entirety is an extremely deep and complex topic. It is completely impractical
+for a single crate to meet all color needs. The goal of this one is to strike a balance,
+providing color capabilities while also keeping things simple and efficient.
+
+The main purpose of this crate is to provide a good set of types for representing colors,
+along with conversions between them and basic manipulations, especially interpolation. A
+major inspiration is the CSS Color Level 4 draft spec; we implement most of the operations
+and strive for correctness.
+
+Simplifications include:
+  * Always using `f32` to represent component values.
+  * Only handling 3-component color spaces (plus optional alpha).
+  * Choosing a fixed, curated set of color spaces for dynamic color types.
+  * Choosing linear sRGB as the central color space.
+  * Keeping white point implicit.
+
+A number of other tasks are out of scope for this crate:
+  * Print color spaces (CMYK).
+  * Spectral colors.
+  * Color spaces with more than 3 components generally.
+  * [ICC] color profiles.
+  * [ACES] color transforms.
+  * Appearance models and other color science not needed for rendering.
+  * Quantizing and packing to lower bit depths.
+
+## Main types
+
+The crate has two approaches to representing color in the Rust type system: a set of
+types with static color space as part of the types, and [`DynamicColor`] in which the
```

Comment:
These links should be evaluated and linked to in the block above (starting with `[libm]: ...`). You can use rust-analyzer to get the future docs.rs links for each item - see https://linebender.org/blog/doc-include/

Reply:
I'm trying to follow those directions and having trouble. If I do this:

```
//! [AlphaColor]: crate::AlphaColor
```

Then I get ``warning: Could not resolve definition of `crate::AlphaColor`.`` If I leave that out, rdme doesn't complain, but I have no idea whether that would resolve in crates.io.

Reply:
Apologies. What I'm trying to indicate is quite subtle.
The section of the linked page I am referring to is the "Getting external documentation links" section.

To make these links resolve properly, we can have a "fallback" docs.rs link in the README, which will be used by any renderer of the GitHub README.

Using cargo-rdme means we are effectively doing the opposite of the trick described in the post.